bin/ebuild.sh

```diff
@@ -547,6 +547,150 @@ if [[ -n ${QA_INTERCEPTORS} ]] ; then
 	unset BIN_PATH BIN BODY FUNC_SRC
 fi
 
+__call-ebuildshell() {
```

Comment:
I'd like to see this function moved to a separate ebuildshell.sh file that's conditionally sourced, so that ebuild.sh is as small as possible. Also, new internal functions should start with triple underscore so that they are automatically filtered by __save_ebuild_env.